api_test.go

```diff
@@ -90,3 +90,56 @@ func TestBezier(t *testing.T) {
 	gproc = proc.Proc
 	proc.Run(t)
 }
+
+func TestRandom(t *testing.T) {
+	old := gproc
+	defer func() {
+		gproc = old
+	}()
+
+	tests := []struct {
+		min  float64
+		max  float64
+		want float64
+	}{
+		{0, 5, 1.4393455630422243},
+		{-5, -0, -0.6158681223759013},
+		{1, 4, 3.3935187588071685},
+		{-4, -1, -3.0145149335211046},
+		{0, 0, 0},
+		{-1, -1, -1},
+		{1, 1, 1},
+	}
+	for _, tt := range tests {
+		t.Run("", func(t *testing.T) {
+			if got := Random(tt.min, tt.max); got != tt.want {
+				t.Errorf("Random() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestRandomGaussian(t *testing.T) {
+	old := gproc
+	defer func() {
+		gproc = old
+	}()
+
+	tests := []struct {
```

Comment:
perhaps add a call to an explicit seed seeding before going through these tests to ensure reproducibility (and robustness against adding new tests exercizing the PRNG) ?

Reply:
changed to explicit seed on a test local proc to make sure that tests do not interfere with each other